### PR TITLE
Feature/for what block

### PR DESCRIPTION
**Pull request by averingood**

### CODE REVIEW

File: src/components/for-what-block/for-what-block.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, {useState} from "react"
```
Comment (vectorincorporated) [inline src/components/for-what-block/for-what-block.js:1]: Use camel case file and folder naming convention for react components (ForWhatBlock/ForWhatBlock.js)

File: src/components/for-what-block/for-what-block.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+import styles from './for-what-block.module.css';
+
+const classNames = require('classnames');
+
+
+const ForWhatBlock = () => {
+    const [isInfoBlock, toggleInfoBlock] = useState(false);
```
Comment (vectorincorporated) [inline src/components/for-what-block/for-what-block.js:9]: For setter in useState hook use the same name of variable with "set" word before. In this case: [isInfoBlock, setIsInfoBlock]

File: src/components/for-what-block/for-what-block.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+    const showInfoBlock = () => {
```
Comment (vectorincorporated) [inline src/components/for-what-block/for-what-block.js:11]: This two function is doing the same things. 
Create function toggleInfoBlock, which will set new value isInfoBlock.
Be careful, that setIsInfoBlock can take callback function with first parameter as previous isInfoBlock value. You should use it to toggle.

File: src/components/for-what-block/for-what-block.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                    For what
+                </h2>
+                <span className='accent-text'>
+                    we use the best technologiies to enhance your product
+                </span>
+            </div>
+
+            <div className={ styles.info }>
+                <div className={classNames( 'accent-text', styles.infoMenu )}>
+                    <div className={styles.infoItem} onMouseOver={showInfoBlock} onMouseLeave={hideInfoBlock}>
```
Comment (vectorincorporated) [inline src/components/for-what-block/for-what-block.js:32]: I see that this divs are repeats. Create component that will receive model and all required callbacks.

**Review by evgeniylovejs — APPROVED** [review]

**Pull request merged**